From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mgdx <leo@magadoux.fr>
Date: Mon, 7 Sep 2026 23:35:12 +0200
Subject: [PATCH] Increment version for search engines data for Startpage

Existing profiles only pick up changes to the prepopulated search
engines data when this version is incremented, otherwise they keep the
list they were created with.
---
 definitions/prepopulated_engines.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/definitions/prepopulated_engines.json b/definitions/prepopulated_engines.json
index 7818713..c713f68 100644
--- a/definitions/prepopulated_engines.json
+++ b/definitions/prepopulated_engines.json
@@ -28,7 +28,7 @@
     // existing data should get a new version. Otherwise, existing data may
     // continue to be used and updates made here will not always appear.
     // Also then run tools/search_engine_choice/generate_search_engine_icons.py.
-    "kCurrentDataVersion": 211
+    "kCurrentDataVersion": 212
   },
 
   // The following engines are included in country lists and are added to the
